microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md

```diff
@@ -283,6 +283,14 @@ Microsoft Managed Desktop recommends adjusting guest invitation settings, since
 
 **Member users and users assigned to specific admin roles can invite guest users including guests with member permissions** should be enabled. For steps, see [Prerequisites for guest accounts](/microsoft-365/managed-desktop/get-ready/guest-accounts).
 
+### Guest user access
+
+Microsoft Managed Desktop recommends adjusting guest user access, since the default setting allows all guest users in your directory to have the same access as members.
+
+**Advisory**
```

Comment:
```suggestion
#### Advisory
```

Reply:
This does not follow the pattern for the other checks. Advisory is not a header but instead should just be bold. I believe **Advisory** is the correct way to show this

Reply:
Thank you for your response. I take your point. Please ignore my suggestion. Many thanks.